de1plus/bluetooth.tcl

```diff
@@ -1373,17 +1462,29 @@ proc de1_ble_handler { event data } {
 
 						de1_connect_handler $handle $address "DE1"
 
-						if {$::de1(scale_device_handle) != 0} {
+						if {$::de1(scale_device_handle) != 0 && $::de1(thermometer_device_handle) != 0 } {
```

Comment:
At a higher level, does it make sense to have a "check for" list somewhere so that the scanner can be stopped, at least at boot? Maybe stop if started from boot (or wake) if _configured_ devices are all found?

I haven't watched in a lot of detail, but it seems that at least DSx dumps the whole set of shot frames and settings onto the BLE queue at load time, often even before the DE1 is connected. Maybe that's a different problem to resolve.